### PR TITLE
Added a note in menu docs

### PR DESCRIPTION
**Pull request by adithyagenie**
Fixes grammyjs/menu#39

### CODE REVIEW

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]
What does it mean that it “does not work?” How can this be fixed? I do not understand what you're trying to document here (and I wrote the plugin)

File: site/docs/plugins/menu.md [hunk 1]
```diff
@@ -340,6 +340,10 @@ bot.command("publish", async (ctx) => {
 
 Payloads also work well together with dynamic ranges.
 
+> Note that using [dynamic buttons](#dynamic-ranges) which generate based on the payload might not work.
```
[the remark this answers is not in the document]
Comment (adithyagenie) [reply site/docs/plugins/menu.md:343]: I meant buttons created by dynamic ranges, that was poor wording. My bad.

**Comment by adithyagenie** [comment]
> What does it mean that it “does not work?”

I meant that it might produce undesirable results, as the menu will be re-rendered partially with some other payload than what was used to render it initially.

I am not really that capable to writing documentations that well. Apologies.

**Comment by KnorpelSenf** [comment]
Do you think #869 works well?

**Comment by adithyagenie** [comment]
Yes, #869 is fine! I'll close this PR now.

**Pull request closed without merging**